.rubocop.yml

```diff
@@ -1,15 +1,16 @@
-inherit_from: .rubocop_todo.yml
-
-inherit_gem:
-  rubocop-govuk:
-    - config/default.yml
-    - config/rails.yml
+inherit_from:
+  - .rubocop_todo.yml
+  - ./lib/rubocop/config/rspec.yml
+  - ./lib/rubocop/config/rails.yml
+  - ./lib/rubocop/config/rake.yml
+  - ./lib/rubocop/config/style.yml
+  - ./lib/rubocop/config/layout.yml
+  - ./lib/rubocop/config/metrics.yml
```

Comment:
I'm not sure that this is the best place for the configuration, but I do think that everything rubocop related should be grouped. Should I move it all to config/rubocop instead? (along with the custom defined cops?)

Reply:
Good question. From my perspective it feels ok to separate them, as they serve different purposes. One is the actual config and feels better suited under `config/rubocop`. The cops can still live as now in `lib` as utils which we require.

Im also ok to keep as is though, but not keen on moving things under `config/rubocop` as the cops feel a bit misplaced there

Reply:
Ok. I think I'll move the configuration only then to config/rubocop